src/recordpage.cpp

```diff
@@ -454,6 +461,12 @@ RecordPage::RecordPage(QWidget* parent)
         if (index != -1)
             ui->compressionComboBox->setCurrentIndex(index);
     }
+
+#ifdef __linux__
```

Comment:
this looks wrong, why do you check the platform? the konsole part could be available on other platforms too after all?

Reply:
As far as I know there is now tail on windows. Since it would not work I disabled it.

Reply:
but its available on freebsd, mac and even on windows potentially one could supply `tail`

i.e.: this is the wrong way to handle this, instead you should only keep the failsafe mode to gracefully handle the situation where `tail` wasn't found (it may even not be available on linux after all)

Reply:
good point. I removed the ifdef and tested both cases (konsolepart not found and tail not found). In both cases the fallback to the textarea worked.